lambda_src/drivers/process_https.py

```diff
@@ -123,10 +124,10 @@ def process_row(
         elif 'headers' in req_auth:
             req_headers.update(req_auth['headers'])
         elif 'body' in req_auth:
-            if json:
-                raise ValueError(f"auth 'body' key and json param are both present")
+            if data:
```

Comment:
keep the json error so that a Snowflake user can't override what's in the secret by passing a `json` header, either